docs/install/amazon-web-services-eks.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Amazon Web Services Elastic Kubernetes Service (AWS EKS)"
+linkTitle: "Amazon Web Services Elastic Kubernetes Service (AWS EKS)"
+---
+
+## Overview
+
+This guide will demonstrate how to install Ondat onto a [Elastic Kubernetes Service](https://aws.amazon.com/eks/) cluster using the [Ondat kubectl plugin](/docs/reference/kubectl-plugin/).
+
+## Prerequisites
+
+* You have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](/docs/prerequisites/) page for more information.
+
+* The following CLI utilities are installed on your local machine and are available in your `$PATH`:
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [kubectl-storageos](/docs/reference/kubectl-plugin/)
+
+* You have a running EKS cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
+
+* Your EKS clusters use [Ubuntu for EKS](https://cloud-images.ubuntu.com/docs/aws/eks/) as the default node operating system with an optimised kernel. For the kernel module `tcm_loop`, the package `linux-modules-extra-$(uname -r)` is additionally required on each of the nodes - this can be installed automatically by adding it to the node's user data as in the example below.
+
+In this example, we have used [eksctl](https://eksctl.io/introduction/) to create a cluster with 3 nodes of size `t3.large` running Ubuntu for EKS in the `eu-west-2` region. We have provided `100 GB` of disk space for each node - note that by default, Ondat will store data locally in the node's file system under the path `/var/lib/storageos` on each node in [hyperconverged mode](/docs/concepts/nodes/#hyperconverged-mode). In a production infrastructure, we would likely create multiple EBS Volumes tweaked for performance or use ephemeral SSD storage and [mount our volumes under data device directories](/docs/concepts/volumes/) with some additions to user data. We would also implement some form of snapshots or backup of these underlying volumes to ensure continuity in a disaster scenario.
+
+```yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: ondat-cluster
+  region: eu-west-2
+
+managedNodeGroups:
+  - name: ondat-ng
+    minSize: 3
+    maxSize: 3
+    instanceType: t3.large
+    ami: ami-0cb2cb474d9e4e075
+    labels: {ondat: node}
+    volumeSize: 100
+    volumeName: /dev/xvda
+    volumeEncrypted: true
+    disableIMDSv1: true
+    overrideBootstrapCommand: |
+      #!/bin/bash
+      mkdir -p /var/lib/storageos
+      echo "/dev/nvme1n1 /var/lib/storageos ext4 defaults,discard 0 1" >> /etc/fstab
+      mkfs.ext4 /dev/nvme1n1
```

Comment:
The volume created from the ng is the root fs. I don't think eksctl supports multiple EBS mounts yet https://github.com/weaveworks/eksctl/issues/1253#issuecomment-1026538351

I'm struggling to see how we are using /dev/nvme1n1 when there is only one EBS that will be nvme0n1 mounted on / as

```
nvme0n1     259:0    0    50G  0 disk
└─nvme0n1p1 259:1    0    50G  0 part /
```

Reply:
Ah-ha, I've tested this and it's not the case - when I specify the volume, it's not mounted to `/` (instead, the AMI defines the volume mounted to `/`). It's just left hanging there unless we mount it - at least on the AMI mentioned.

https://github.com/weaveworks/eksctl/pull/4725 enables additional volume mappings over and above the default 1 extra volume!

Reply:
Interesting, that is bleeding edge functionality 😃. I see what you mean. On the version I'm running of eksctl is not there (30 days old release). Given the fact, that the functionality was released 11 days ago in the eksctl 0.83.0, I would suggest to add a disclaimer or prereq of >=0.83.0 version.